### PR TITLE
Add ui.multiselect via single package.json w/ git auto-update

### PR DESCRIPTION
**Pull request by maruilian11**
Pull request for issue: #11646 
@cdnjs/library-reviewer This lib is `jqgrid`'s dependent lib. Please help me review this PR ASAP, thanks!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/michael/multiselect
 * Official website (optional, not the repository): http://www.quasipartikel.at/multiselect
 * NPM package url (optional):
 * License and its reference: MIT, https://github.com/michael/multiselect/blob/master/MIT-LICENSE.txt
 * GitHub / Bitbucket popularity (required):
   Watch 24
   Star 562
   Fork 214

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/ui.multiselect/0.3.1/js/ui.multiselect.js [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+				return false;
+		})
+		.keyup(function() {
+			that._filter.apply(this, [that.availableList]);
+		});
+	}
+});
+		
+$.extend($.ui.multiselect, {
+	locale: {
```
Comment (sufuf3) [inline ajax/libs/ui.multiselect/0.3.1/js/ui.multiselect.js]: I wonder to know that should we add files under `js/locale`?
Thank you.

**Comment by maruilian11** [comment]
@sufuf3 Thanks for that.

File: ajax/libs/ui.multiselect/0.3.1/js/ui.multiselect.js [hunk 2]
```diff
@@ -0,0 +1,336 @@
+/*
+ * jQuery UI Multiselect
+ *
+ * Authors:
+ *  Michael Aufreiter (quasipartikel.at)
+ *  Yanick Rochon (yanick.rochon[at]gmail[dot]com)
+ * 
+ * Dual licensed under the MIT (MIT-LICENSE.txt)
+ * and GPL (GPL-LICENSE.txt) licenses.
```
Comment (extend1994) [inline ajax/libs/ui.multiselect/0.3.1/js/ui.multiselect.js]: We might need to confirm if there is GPL license used.

File: ajax/libs/ui.multiselect/0.3.1/js/ui.multiselect.js [hunk 3]
```diff
@@ -0,0 +1,336 @@
+/*
+ * jQuery UI Multiselect
+ *
+ * Authors:
+ *  Michael Aufreiter (quasipartikel.at)
```
Comment (extend1994) [inline ajax/libs/ui.multiselect/0.3.1/js/ui.multiselect.js]: Maybe add `url` info for this author?

File: ajax/libs/ui.multiselect/0.3.1/js/ui.multiselect.js [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+ *  Yanick Rochon (yanick.rochon[at]gmail[dot]com)
```
Comment (extend1994) [inline ajax/libs/ui.multiselect/0.3.1/js/ui.multiselect.js]: There is another author, let's take him as one of the author?

**Comment by sufuf3** [comment]
Because the [README](https://github.com/michael/multiselect#jquery-ui-multiselect) show
> This repository is no longer actively maintained by the author.

Could we still waiting for the answer?
How about updating the info after this PR be merged and get the confirm?

@maruilian11 @extend1994 What do you think?
Many thanks.

**Comment by sashberd** [comment]
@extend1994 @PeterDaveHello Please help me to rebase to lates master in order to finish this PR. Thanks

File: ajax/libs/ui.multiselect/0.3.1/js/ui.multiselect.js (hunk 2 above)
Comment (sashberd) [reply ajax/libs/ui.multiselect/0.3.1/js/ui.multiselect.js]: @extend1994 According to this [link](https://github.com/michael/multiselect/tree/next) and this [page](http://quasipartikel.at/multiselect_next/)
The widget under MIT and GPL v2.0 license

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Comment by sashberd** [comment]
@xtuc @MattIPv4 FYR

File: ajax/libs/ui.multiselect/package.json [hunk 5]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "ui.multiselect",
+  "description": "Another attempt of a sortable, searchable multiple select widget",
+  "filename": "js/ui.multiselect.min.js",
```
Comment (MattIPv4) [inline ajax/libs/ui.multiselect/package.json]:
```suggestion
  "filename": "js/ui.multiselect.js",
```
This file doesn't appear to exist in the latest tagged version, just the unminified version: https://github.com/michael/multiselect/tree/v0.3.1/js

Comment (sashberd) [reply ajax/libs/ui.multiselect/package.json]: Updated

**Pull request merged**